packages/context/src/objects/objects.ts

```diff
@@ -278,6 +275,13 @@ export type ActionInputType =
     | 'Time'
     | 'User';
 
+export type DocumentType = 'Portal' | 'Private' | 'Public';
+
+export type DocumentMetadata = {
```

Comment:
Document metadata is open-ended.  As far as the platform is concerned, this is just `Record<string, string>`.